hack/jenkins/build.sh

```diff
@@ -62,7 +62,11 @@ git clean -fdx
 # docker images -q | xargs -r docker rmi
 
 # Build
-go run ./hack/e2e.go -v --build
+if [[ "${KUBE_FASTBUILD:-}" == "true" ]]; then
```

Comment:
The release tooling [calls](https://github.com/kubernetes/release/blob/master/anago#L414) `make release`.